stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -30,6 +30,7 @@ spec:
 {{ toYaml .Values.controller.podLabels | indent 8 }}
         {{- end }}
     spec:
+      dnsPolicy: {{ .Values.controller.dnsPolicy | default "ClusterFirst" }}
```

Comment:
Please add the default to `values.yaml` instead of using the `default` function.